tests/API_smoke_test.postman_collection.json

```diff
@@ -464,6 +464,367 @@
+								"v1",
+								"activities",
+								"pagingtoken.json"
+							],
+							"query": [
+								{
+									"key": "sinceDatetime",
+									"value": "{{pastHour}}"
+								},
+								{
+									"key": "access_token",
+									"value": "{{accessToken}}"
+								}
+							]
+						},
+						"description": "returning paging token for last hour in order to get lead changes in the following request"
+					},
+					"response": []
+				},
+				{
+					"name": "Verify campaign emails being sent",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "9cbc4390-0752-4a23-b367-ea827ffd6f87",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "cacd1abb-0016-4dbd-875d-fa5dd33c1a4e",
+								"exec": [
+									"const assert = eval(globals.loadAsserts);",
+									"assert.common(200, 1000, false, null, \"\");",
+									"",
+									"pm.test(\"Marketo has delivered emails within the last hour\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.result).to.be.an('array').that.is.not.empty;",
+									"});",
+									"",
+									"   "
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "https://248-ISZ-536.mktorest.com/rest/v1/activities.json?access_token={{accessToken}}&nextPageToken={{nextPageToken}}&activityTypeIds=7&=&&",
+							"protocol": "https",
+							"host": [
+								"248-ISZ-536",
+								"mktorest",
+								"com"
+							],
+							"path": [
+								"rest",
+								"v1",
+								"activities.json"
+							],
+							"query": [
+								{
+									"key": "access_token",
+									"value": "{{accessToken}}"
+								},
+								{
+									"key": "nextPageToken",
+									"value": "{{nextPageToken}}"
+								},
+								{
+									"key": "activityTypeIds",
+									"value": "7"
+								},
+								{
+									"key": "",
+									"value": ""
+								},
+								{
+									"key": null,
+									"value": null
+								},
+								{
+									"key": "",
+									"value": null
+								}
+							]
+						},
+						"description": "Verifying that marketo has attempted to send emails within the **last hour**."
+					},
+					"response": []
+				}
+			],
+			"description": "A series of tests designed around making sure new clinicians and DSAs are being connected via shoreline to our marketing campaigns.",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "1aa215b0-df87-4014-b78a-0236be1b78af",
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"id": "07c5ca86-781b-4d89-8ac9-296f79157d76",
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				}
```

Comment:
these appear to just be empty??

Reply:
fixed

Reply:
Looks like postman puts this after every folder. its also around line 443 for the smoke test folder